.gitignore

```diff
@@ -19,3 +19,15 @@ tools/elf2dol
 *.cp
 .vscode
 .pragma
+tools/lmgr8c.dll
+tools/lmgr326b.dll
+ldscript.lcf
+asmdiff.sh
+.clang-format
+SpongeBob SquarePants - Battle for Bikini Bottom.7z
+ldscript.lcf
+asmdiff.sh
+.clang-format
+ldscript.lcf
+asmdiff.sh
+.clang-format
```

Comment:
I don't know if a program generated this stuff or if you handwrote it, but none of this should be here. A lot of those things are scripts that we do want tracked by git if they are changed.